packages/contracts/src/predeploys.ts

```diff
@@ -16,9 +16,10 @@ export const predeploys = {
   L2StandardTokenFactory: '0x4200000000000000000000000000000000000012',
   BVM_L1BlockNumber: '0x4200000000000000000000000000000000000013',
 
-  // We're temporarily disabling BVM_ETH because the jury is still out on whether or not ETH as an
+  // We're temporarily disabling BVM_ETH because the jury is still out on whether ETH as an
```

Comment:
why not  "We're temporarily disabling BVM_BIT because the jury is still out on whether or not BIT as an ERC20 is desirable"

Reply:
done